ios/FluentUI/Pill Button Bar/PillButtonBar.swift

```diff
@@ -220,25 +160,82 @@ open class PillButtonBar: UIScrollView, FluentUIWindowProvider {
         buttons[index].isEnabled = true
     }
 
-    open override func layoutSubviews() {
-        super.layoutSubviews()
+    public typealias TokenType = PillButtonBarTokens
```

Comment:
I know I was just told in the PillButton PR I didn't need this, but Xcode strongly disagreed this time. Not sure what the difference is? I'm able to build PillButtons without the typealias, but PillButtonBar won't have it.